src/components/cards/CardHeader.tsx

```diff
@@ -1,15 +0,0 @@
-import React from "react";
-
-interface NewCardHeaderProps {
-  cardName: string;
```

Comment:
You could argue that you use `children` here instead. If so then you would just do `<NewCardHeader>MyCardName</NewCardHeader>` instead of `<NewCardHeader cardName="">`. 

Not much of a difference really. If you plan on passing in icons or other html elements, then use children. If not, keep with the strings.